packages/manager/modules/nasha/src/components/edit-name/edit-name.template.html

```diff
@@ -19,7 +19,7 @@
             class="oui-input"
             data-ng-model="$ctrl.model.name"
             data-ng-pattern="$ctrl.namePattern"
-            data-mismatch="{{:: $ctrl.model.name }}"
+            data-forbid="{{:: $ctrl.forbid }}"
```

Comment:
Get rid of this useless `forbid` property and use `model.name` here and `nasha.customName` for the error message translation instead